athens3/binary_list.json

```diff
@@ -99,6 +99,10 @@
     {
       "download_url": "https://github.com/zeta-chain/node/releases/download/v19.0.0/zetacored-linux-amd64",
       "binary_location": "cosmovisor/upgrades/v19/bin/zetacored"
+    },
+    {
+      "download_url": "https://github.com/zeta-chain/node/releases/download/v20.0.1/zetacored-linux-amd64",
```

Comment:
we should skip straight to https://github.com/zeta-chain/node/releases/tag/v20.0.2